cypress/integration/entitybrowser_spec.js

```diff
@@ -1,59 +1,88 @@
 import helpers from '../helpers/helpers'
 
+const getPkFromUrl = url => url.substring(url.lastIndexOf('/') + 1)
+
+const createUser = (firstname, lastname) => {
+  return cy.request({
+    url: `${Cypress.env('BACKEND_URL')}/nice2/rest/entities/User`,
+    method: 'POST',
+    json: true,
+    body: {model: 'User', key: null, paths: {firstname, lastname, relGender: {display: 'Weiblich', key: '2'}}}
+  }).then(response => {
+    const location = response.headers.location
+    return getPkFromUrl(location)
+  })
+}
+
+const deleteUser = pk => {
+  return cy.request({
+    url: `${Cypress.env('BACKEND_URL')}/nice2/rest/entities/User/${pk}`,
+    method: 'DELETE'
+  })
+}
+
+const visitEntityBrowser = (suffix = '') => {
+  cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}${suffix}`)
+}
+
 describe('Entity Browser', function() {
   before(() => {
     cy.login()
   })
 
-  describe('ListView', () => {
-    beforeEach(() => {
-      cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}`)
-    })
+  describe('Create', () => {
+    it.only('should navigate to create view wit NEW button', function() {
```

Comment:
`function() {...` -> why not arrow function?

Reply:
cypress does not fully support arrow functions in combination with this.* for aliases. 
https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Aliases

But we dont need that anyways --> i change all  to arrow syntax